packages/template-retail-react-app/app/utils/responsive-image.js

```diff
@@ -18,32 +18,56 @@ const getBreakpointLabels = (breakpoints) =>
         .map(([key]) => key)
 
 const {breakpoints: defaultBreakpoints} = theme
+const defaultDensityFactors = [
```

Comment:
Since these are constant values should we use uppercase?
```suggestion
const DEFAULT_DENSITY_FACTORS = [
```

Reply:
Please make this upper case

Reply:
That's an internal-only constant. None of the other internal constants is using the ancient naming scheme that's used for publicly exposed constants. So I'll stick to that name.